front-end/src/components/KeyMeasureDiagnostic/Inputs/BaseCheckboxes.vue

```diff
@@ -3,9 +3,9 @@
     <legend>{{legend}}</legend>
 
     <div class="checkboxes">
-      <div v-for="option in options" :key="option.inputId" class="checkbox">
-        <input :id="option.inputId" :checked="modelValue.includes(option.value)" @input="updateInput(option.value, $event)" type="checkbox" >
-        <label :for="option.inputId"><p>{{option.label}}</p></label>
+      <div v-for="key in Object.keys(options)" :key="key" class="checkbox">
```

Comment:
Un petit chose: je pense qu'on peut faire `v-for="(option, key) in options"`

Reply:
Ha oui en effet, bien vu.